### ISSUE TITLE
[Bug]: Starting the Command Line fails with `FileNotFoundError`

### ISSUE THREAD

**Issue by gcarq** [bottlesdevs/Bottles#3488]
### Describe the bug

The "Command Line" tool fails because of `FileNotFoundError: [Errno 2] No such file or directory` within a newly created bottle. No settings have been changed and the bottle has been created with the defaults (`soda-9.0-1` runner).

### To Reproduce

1. Open Bottles
2. Navigate to Settings -> Advanced -> Bottles Directory and change the directory to some non-default location
3. Click on "Create new Bottle"
4. Choose a name, set the environment to Gaming and click on Create
5. Open The bottle after it has been created
6. Navigate to Tools and click on "Command Line"

Expected:
The command line opens

Actual:
Nothing happens

### Package

Flatpak from Flathub

### Distribution

Gentoo

### Debugging Information

```shell
Official Package: true
Version: '51.13'
DE/WM: plasma
Display:
    X.org: true
    X.org (port): :0
    Wayland: true
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.10.6-zen1
Disk:
    Total: 33528705024
    Free: 33528553472
RAM:
    MemTotal: 62.5GiB
    MemAvailable: 53.8GiB
Bottles_envs: null
```

### Troubleshooting Logs

```shell
Sep 02 21:35:10 gentoo3 flatpak[71177]: 21:35:10 (INFO) Using Wine Command Line -- launch_terminal 
Sep 02 21:35:10 gentoo3 flatpak[71177]: 21:35:10 (INFO) Command: easyterm.py -d -p "#00ffff #2b2d2e" -c 'bash -c '"'"'/home/michi/.var/app/com.usebottles.bottles/data/bottles/runners/soda-9.0-1/bin/wine64 cmd'"'"'' 
Sep 02 21:35:10 gentoo3 easyterm.py[73206]: Locale not supported by C library.
                                                    Using the fallback 'C' locale.
Sep 02 21:35:10 gentoo3 flatpak[73206]: Traceback (most recent call last):
Sep 02 21:35:10 gentoo3 flatpak[73206]:   File "/app/bin/easyterm.py", line 12, in <module>
Sep 02 21:35:10 gentoo3 flatpak[73206]:     CONF_DEF_CWD = os.getcwd()
Sep 02 21:35:10 gentoo3 flatpak[73206]:                    ^^^^^^^^^^^
Sep 02 21:35:10 gentoo3 flatpak[73206]: FileNotFoundError: [Errno 2] No such file or directory
```

**Comment by lob19** [comment]
This happened to me right now.

Seems to be related to the global bottles directory path (I set an external device as my directory)
If i changed it back to the default location (`.var/app/com.usebottles.bottles/data/bottles/bottles/`) and i recreate the bottle, the error disappears.

**Comment by gcarq** [comment]
@lob19 good catch. I modified the default location as well and it seems related to it. I've edited the original post to reflect this

**Comment by ddenev** [comment]
I have the same problem with v51.16. Fixed it by also setting the working directory in the bottle itself:
![Screenshot From 2024-12-14 17-30-45](https://github.com/user-attachments/assets/31ec966c-70a2-4bf5-8552-c031e64538cf)

Now I can open the command line.

**Comment by ddenev** [comment]
Spoke too soon.

You also need to:
1. Allow Bottles flatpak to access the external folder (e.g. via Flatseal)
2. Do the bottle config directory update upon every start of Bottles (as it resets to the default one)

Definitely needs a fix! :-)

**Comment by gcarq** [comment]
This is still an issue with version 51.21
![Image](https://github.com/user-attachments/assets/92392e74-afb6-4382-983f-5924a7074d86)
